packages/vscode/src/extension.ts

```diff
@@ -66,9 +67,13 @@ export async function activate(context: ExtensionContext): Promise<void> {
   const isDebugOrTest = isDebugOrTestSession()
 
   const rootPath = workspace.rootPath
+
   if (rootPath) {
     watcher = chokidar.watch(
-      path.join(rootPath, '**/node_modules/.prisma/client/index.d.ts'),
+      [
+        path.join(rootPath, '**/node_modules/.prisma/client/index.d.ts'),
+        '$HOME/.vscode/extensions',
```

Comment:
This change looks unrelated, right? It adds "extensions" folder to watch list but the PR doesn't use it.

Reply:
Yes, I'll remove that.